test/bridge.jl

```diff
@@ -101,28 +123,27 @@ MOIU.@model NoRSOCModel () (EqualTo, GreaterThan, LessThan, Interval) (Zeros, No
         @test MOIB.bridge(bridgedmock, c) isa MOIB.RSOCtoPSDCBridge
     end
 
+    @testset "Combining two briges" begin
+        fullbridgedmock = MOIB.fullbridgeoptimizer(mock, Float64)
+        mock.optimize! = (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock, [1, 1, 0, 1, 1, 0, 1, √2])
+        config = MOIT.TestConfig()
+        MOIT.rootdett1vtest(fullbridgedmock, config)
+        MOIT.rootdett1ftest(fullbridgedmock, config)
+        @test !MOI.canget(fullbridgedmock, MOI.ConstraintDual(), MOI.ConstraintIndex{MOI.VectorAffineFunction{Float64}, MOI.RootDetConeTriangle})
```

Comment:
Why do you expect to not be able to get the constraint dual? Because it's not implemented in the bridge?

Reply:
Yes, it's not yet implemented.

Reply:
Better to add a comment to that effect then